229292.py

```
@@ -0,0 +1,93 @@
+# Function with no return value
+def calculate_sum(a, b):
+    sum = a + b
+    print(sum)  # Only prints the result, doesn't return it
+
+result = calculate_sum(5, 3)
+print(result)  # None, because the function doesn't return anything
+
+# Poor use of exception handling
+def read_file(file_path):
+    try:
+        file = open(file_path, 'r')  # No context manager
```

Ruff (0.8.2)

Use a context manager for opening files

(SIM115)